internal/app/singularity/cache_clean_linux.go

```diff
@@ -65,10 +65,9 @@ func CleanCache(cacheType string) error {
 		err := cache.Clean()
 		return err
 	default:
-		sylog.Fatalf("Not a valid type: %v", cacheType)
-		os.Exit(2)
+		// The caller checks the returned error and will exit as required
+		return fmt.Errorf("not a valid type: %s", cacheType)
 	}
-	return nil
```

Comment:
Why did you remove this line?

Reply:
Yes the linter complained because it is a non-reachable line: all cases of the switch are calling return, including default.

Reply:
Okay :+1: